### ISSUE TITLE
This Warp is Not Safe Right Now. Try Again Later.

### ISSUE THREAD

**Issue by BirkhoffLee** [tastybento/ASkyBlock-Bugs-N-Features#230]
I wanna share my personal experience of how I solved this section of the FAQ: `When I do /is it says "This Warp is Not Safe Right Now. Try Again Later." What Is Wrong?`

A few days ago my server suddenly crashed, I saw the last moment when it was still running, a player created a new island and started playing. A few seconds later, server crashed cuz of an unknown reason. When the server restarted and the same player tried to tp to his island, it says `This Warp is Not Safe Right Now. Try Again Later.`.

I looked around and found that his island was still exists(registered in askyblock), but blocks was lost. I realized that the server didn't have time to save worlds data. So doing `/asadmin delete <player>` solves the problem.

TL;DR: add the following reason in `When I do /is it says "This Warp is Not Safe Right Now. Try Again Later." What Is Wrong?` FAQ section so people can quick know what's going on.
```
The server crashed and the world data was corrupt. 
```

**Comment by tastybento** [comment]
Thanks - added!
